cannot call non-const fn `<Self as PartialEq<Rhs>>::eq` in constant functions

I'm trying to compile `const_units` using nightly but I keep getting this error:

> error[E0015]: cannot call non-const fn `<Self as PartialEq<Rhs>>::eq` in constant functions
>    --> /home/elpiel/.cargo/registry/src/index.crates.io-6f17d22bba15001f/const_ops-0.2.2/src/lib.rs:117:15
>     |
> 117 |         !self.eq(other)
>     |               ^^^^^^^^^
>     |
>     = note: calls in constant functions are limited to constant functions, tuple structs and tuple variants
>     = help: add `#![feature(effects)]` to the crate attributes to enable
> 
> For more information about this error, try `rustc --explain E0015`.
> error: could not compile `const_ops` (lib) due to 1 previous error
> 

I tried adding `#![feature(effects)]` to const_units but didn't work. Maybe it should be added to this crate instead?

--- a/README.md
+++ b/README.md
@@ -1,5 +1,8 @@
 # A const-ready version of core::ops / std::ops arithmetic traits
 
+Requires: Rust **nightly**
+
 All traits are the same as in std / core but prefixed with the #[const_trait] attribute to allow usage in const situations.
 
 This crate is designed in such a way that, as soon as the #[const_trait] attribute is used in core / std all imports of this crate can simply be deleted.
+

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,0 +1,5 @@
+[toolchain]
+channel = "nightly"
+# components = [ "rustfmt", "rustc-dev" ]
+# targets = [ "wasm32-unknown-unknown", "thumbv2-none-eabi" ]
+# profile = "minimal"

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,5 +1,2 @@
 [toolchain]
 channel = "nightly"
-# components = [ "rustfmt", "rustc-dev" ]
-# targets = [ "wasm32-unknown-unknown", "thumbv2-none-eabi" ]
-# profile = "minimal"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,7 +1,6 @@
-#![feature(const_trait_impl, const_mut_refs, const_fn_floating_point_arithmetic)]
+#![feature(const_trait_impl, const_mut_refs, const_fn_floating_point_arithmetic, effects)]
 
 mod impls;
-pub use impls::*;
 
 #[const_trait]
 pub trait Add<Rhs = Self> {

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,6 +1,7 @@
 #![feature(const_trait_impl, const_mut_refs, const_fn_floating_point_arithmetic, effects)]
 
 mod impls;
+pub use impls::*;
 
 #[const_trait]
 pub trait Add<Rhs = Self> {

